app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,76 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+    <%= t('doc_auth.headings.combined_upload') %>
+<% end %>
+
+<p>
+    <%= t('doc_auth.info.combined_upload') %>
+</p>
+
+<hr class="margin-y-4" />
+
+<div class="grid-row">
+    <div class="grid-col-12 tablet:grid-col-3">
+        <%= image_tag(
+              asset_url('idv/phone-icon.svg'),
+              alt: t('image_description.camera_mobile_phone'),
+              width: 100,
```

Comment:
In the Figma design these illustrations these are 88x88, not 100x100.

Reply:
Related: we have a linting rule that requires us to specify image dimensions explicitly in the erb files. Is that something we really want? Couldn't we rely on the containing grid?

Reply:
It is something we want. Related resource: https://web.dev/optimize-cls/#images-without-dimensions

Technically it may not always be an issue if the grid will reserve the space anyways (though this definitely ties to [my other comment](https://github.com/18F/identity-idp/pull/7849#discussion_r1110180655)), but I still think it's a good default to have, since grid placement of images isn't all that common, and I the potential downsides either don't exist or are at least not all that concerning.

Reply:
Coincidentally, there used to be pretty severe & noticeable layout shifts on the IdV upload step, which was part of my own motivation for adding the linter.

Reply:
Got it. I've adjusted these to the Figma sizing here https://github.com/18F/identity-idp/pull/7849/commits/22ee1f2739d7e5f3c66f3dec50aed06318f7be86